recipe/meta.yaml

```diff
@@ -17,25 +17,24 @@ build:
     script: python setup.py install --single-version-externally-managed --record=record.txt
     entry_points:
         - celery = celery.__main__:main
-        - celeryd = celery.__main__:_compat_worker
-        - celerybeat = celery.__main__:_compat_beat
-        - celeryd-multi = celery.__main__:_compat_multi
 
 requirements:
     build:
         - python
         - setuptools
-        - pytz
-        - billiard >=3.3.0.23,<3.4
-        - kombu >=3.0.34,<4.0
     run:
         - python
         - pytz
```

Comment:
Per [`default.txt`](https://github.com/celery/celery/blob/master/requirements/default.txt), seems like this should be `pytz >dev`. (That might be infeasible with conda, though.)